UITests/TrackingProtectionTests.swift

```diff
@@ -72,12 +70,17 @@ class TrackingProtectionTests: KIFTestCase, TabEventHandler {
         wait(for: [setup], timeout: 5)
 
         let clear = self.expectation(description: "clearing")
-        ContentBlockerHelper.clearWhitelist() { clear.fulfill() }
+        ContentBlocker.shared.clearWhitelist() { clear.fulfill() }
         waitForExpectations(timeout: 2, handler: nil)
+
+        NotificationCenter.default.addObserver(forName: .didChangeContentBlocking, object: nil, queue: nil) { arg in
```

Comment:
Are the tests self contained? If so moving them into the same folder as the content blocking source would make sense.